api/src/main/java/jakarta/persistence/criteria/TemporalField.java

```diff
@@ -0,0 +1,27 @@
+package jakarta.persistence.criteria;
```

Comment:
this file needs a license header

Reply:
Well, sure, but I'm also still waiting for feedback on the while design of it. (See my "Questions" above.)

Reply:
I'm fine with the package you propose, unless there is some hidden intent to use this class also on other places - that is the only case I can think of where `jakarta.persistence` would be better